container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/docker_config_storage_creator/docker_config_storage_creator.go

```diff
@@ -0,0 +1,175 @@
+package docker_config_storage_creator
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/docker/docker/api/types/registry"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/docker_manager"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// We use this image and version because we already are using this in other projects so there is a high probability
+	// that the image is in the local machine's cache
+	creatorContainerImage = "alpine:3.17"
+	creatorContainerName  = "kurtosis-docker-config-storage-creator"
```

Comment:
Not something that is blocking this PR but thinking out loud for the future - we have a few of these "config creation" containers that get spun up in the kurtosis lifecycle (fluentbit log creator, docker config, github auth, logs aggregator config) - each one of these gets spun up as its own container and then removed. It would reduce overhead if all these operations were performed on one container (all of them are just rlly mounting to a volume and running a command).

Reply:
Makes sense